manifests/application.pp

```diff
@@ -65,14 +70,7 @@
     group   => $::foreman::group,
     mode    => '0640',
     content => template('katello/katello.yaml.erb'),
-    notify  => [Class['foreman::service', 'foreman::plugin::tasks'], Foreman::Rake['db:seed']],
-  }
-
-  if $enable_ostree {
-    package { $rubygem_katello_ostree:
-      ensure => installed,
-      notify => [Class['foreman::service', 'foreman::plugin::tasks'], Foreman::Rake['apipie:cache:index']],
-    }
+    notify  => [Class['foreman::service', 'foreman::plugin::tasks'], Foreman::Rake['db:seed'], Foreman::Rake['apipie:cache:index']],
```

Comment:
If you depend on theforeman-foreman >= 9.0.0 then you can leave out the tasks notification since the module handles that itself.

Reply:
oh i missed this comment, let me do that.